robot.py

```
@@ -1,127 +1,30 @@
+nonebot.load_plugin("src.module.maintain")
+nonebot.load_plugin("src.module.cron")
+nonebot.load_plugin("src.module.tool.how_to_cook")
+nonebot.load_plugin("src.module.interact")
+nonebot.load_plugin("src.module.cp.nk")
+nonebot.load_plugin("src.module.cp.atc")
+nonebot.load_plugin("src.module.cp.contest_manual")
+if __name__ == "__main__":
+    # 注册模块版本信息
+    for plugin in get_loaded_plugins():
+        if plugin.module_name.startswith("src"):
+            getattr(importlib.import_module(plugin.module_name),"register_module")()
```

Ruff (0.12.2)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)